docroot/sites/registrar.uiowa.edu/modules/registrar_core/src/Controller/AcademicCalendarController.php

```diff
@@ -210,7 +210,7 @@ private function fetchAndProcessCalendarData($start, $end, $categories, $steps =
     $sessions = ((int) $steps === 0) ? [$current] : $this->maui->getSessionsRange($current->id, max(1, $steps));
 
     if ($includePastSessions) {
-      $pastSessions = array_slice($this->maui->getSessionsRange($current->id, -$steps - 1), 0, $steps);
+      $pastSessions = array_slice($this->maui->getSessionsRange($current->id, -$steps - 1), -$steps - 1, $steps + 1);
```

Comment:
It would be great to include a comment here explaining what is happening because I don't think it is obvious from just looking at this why it is necessary.